src/main/java/ita/softserve/course_evaluation/controller/FeedbackRequestController.java

```diff
@@ -114,4 +115,19 @@ public void deleteFeedbackRequest(@PathVariable Long id) {
 				ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null) :
 				ResponseEntity.status(HttpStatus.OK).body(feedbackRequestService.getFeedbackRequestByCourseIdOnly(id));
 	}
+
+	@ApiOperation(value = "Get all feedbackrequests by course id with feedbacks of student id")
+	@ApiResponses(value = {
+			@ApiResponse(code = 200, message = HttpStatuses.OK, response = List.class),
+			@ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+			@ApiResponse(code = 403, message = HttpStatuses.FORBIDDEN),
+	})
+	@GetMapping("/course/{idc}/{ids}")
+	public ResponseEntity<List<StudentFeedbackRequestDto>> getFeedbackRequestByCourseIdAndStudentId(@PathVariable long idc, @PathVariable long ids) {
```

Comment:
maybe it would be better named url  @GetMapping("/course/{course_id}/student/{student_id}")